### PR TITLE
HPE Solution for RedHat OpenShift Container Platform 4.14

### PR DESCRIPTION
**Pull request by jaganhpe**

### CODE REVIEW

**Comment by Rvbh90** [comment]
everything looks good from my end

File: DL-LTI-Openshift/coreos_BareMetalworker_nodes/Readme.md
Comment (srihithavempallihpe) [inline DL-LTI-Openshift/coreos_BareMetalworker_nodes/Readme.md:1]: @Rvbh90 Rakesh, can you update what changes we should do like adding mac address in input file under ocp worker nodes section at line no:8

File: DL-LTI-Openshift/coreos_BareMetalworker_nodes/hosts
Comment (srihithavempallihpe) [inline DL-LTI-Openshift/coreos_BareMetalworker_nodes/hosts:1]: @Rvbh90 can you please eerase ip address and add duplicate ips or in this format 172.X.X.X

File: DL-LTI-Openshift/coreos_BareMetalworker_nodes/playbooks/roles/deploy_ipxe_ocp/profile/tasks/main.yml
Comment (srihithavempallihpe) [inline DL-LTI-Openshift/coreos_BareMetalworker_nodes/playbooks/roles/deploy_ipxe_ocp/profile/tasks/main.yml:1]: - name: copy boot images to match 
 
 this task is commented, please check once.

File: DL-LTI-Openshift/coreos_kvmworker_nodes/Readme.md
Comment (srihithavempallihpe) [inline DL-LTI-Openshift/coreos_kvmworker_nodes/Readme.md:1]: is this coreos_kvmworker_nodes folder required only readme and input file is there

File: DL-LTI-Openshift/coreos_BareMetalworker_nodes/hosts
Comment (Rvbh90) [reply DL-LTI-Openshift/coreos_BareMetalworker_nodes/hosts:1]: removed the hosts IP in the hosts file. we can see the IP's as 172.X.X.X

File: DL-LTI-Openshift/coreos_BareMetalworker_nodes/playbooks/roles/deploy_ipxe_ocp/profile/tasks/main.yml
Comment (Rvbh90) [reply DL-LTI-Openshift/coreos_BareMetalworker_nodes/playbooks/roles/deploy_ipxe_ocp/profile/tasks/main.yml:1]: uncommented the code

File: DL-LTI-Openshift/coreos_kvmworker_nodes/Readme.md
Comment (Rvbh90) [reply DL-LTI-Openshift/coreos_kvmworker_nodes/Readme.md:1]: I can see all the required folders reflecting

**Review by srihithavempallihpe — APPROVED** [review]

**Review by Rvbh90 — APPROVED** [review]

**Pull request merged**